### ISSUE TITLE
 Fullcalendar v4 will be supported after the beta ?  

### ISSUE THREAD

**Issue by MattiCerio** [philippfrenzel/yii2fullcalendar#139]

**Comment by MattiCerio** [comment]
From [Fullcalendar Blog Article](https://fullcalendar.io/blog/2019/03/v4-officially-released)
A year after the project first began, version 4 is finally released. A tremendous amount of work went into it. To summarize the advancements over v3:

- jQuery has been removed
- Moment has been removed and time zones have been rethought
- The API is more modern
- Functionality is broken into lightweight plugins
- Plugins have been added for recurring events, Moment, and Luxon
- Touch interactions and scrolling work better
- Lots of internal refactoring that has already made development much faster

[V4 Release Notes and Upgrade Guide](https://fullcalendar.io/docs/upgrading-from-v3)

**Comment by philippfrenzel** [comment]
yes will migrate to the new version - release planned for easter

**Comment by snickom** [comment]
New version 4.0.2 doesn't have correct paths to bower files. See:

[Fullcalendar v4.0.2](https://github.com/fullcalendar/fullcalendar/tree/v4.0.2/dist)

and your [AssetBundle](https://github.com/philippfrenzel/yii2fullcalendar/blob/master/CoreAsset.php)
and moment package is not installed, you must require it in composer.json.

**Comment by MattiCerio** [comment]
Moment JS dependency has been removed, i suggest to remove the MomentAsset dependency from CoreAsset

**Comment by philippfrenzel** [comment]
Thanks, will remove it

**Comment by MattiCerio** [comment]
In v4 bower paths are changed because the functionalities were splitted in plugins.
Loading scripts : https://fullcalendar.io/docs/initialize-globals
Plugin Index: https://fullcalendar.io/docs/plugin-index

**Comment by dsdobrzynski** [comment]
@philippfrenzel Is there an update on this?
